src/Tizen.NUI/src/internal/FrameBroker/FrameBrokerBase.cs

```diff
@@ -312,17 +312,21 @@ private PropertyBuffer CreateQuadPropertyBuffer()
 
             TexturedQuadVertex[] texturedQuadVertexData = new TexturedQuadVertex[4] { vertex1, vertex2, vertex3, vertex4 };
 
-            int length = Marshal.SizeOf(vertex1);
-            IntPtr pA = Marshal.AllocHGlobal(checked(length * 4));
+            int size = Marshal.SizeOf(vertex1);
+            IntPtr pA = Marshal.AllocHGlobal(size * texturedQuadVertexData.Length);
```

Comment:
```suggestion
            IntPtr pA = Marshal.AllocHGlobal(checked(size * texturedQuadVertexData.Length));
```


Don't forget to use `checked` keyword, which will resolve the svace issue what we resolved before.